src/db_models/CampusHoursBase.py

```diff
@@ -0,0 +1,17 @@
+from sqlalchemy import Column, Integer, String
+from sqlalchemy.ext.declarative import declared_attr
+
+
+class CampusHoursBase(object):
+    @declared_attr
+    def __tablename__(cls):
+        class_name = cls.__name__  # e.g. TrilliumHour
+        table_name = class_name[0].lower() + class_name[1:] + "s"  # e.g. trilliumHours
+        return table_name
+
+    date = Column(String, nullable=False)
+    event_description = Column(String, nullable=False)
+    event_summary = Column(String, nullable=False)
+    end_time = Column(String, nullable=False)
+    id = Column(Integer, nullable=False, primary_key=True)
```

Comment:
It's normally convention to have `id` always be the first field of any model, but it's up to @meganle